src/main/java/javax/validation/ConstraintValidatorContext.java

```diff
@@ -333,6 +333,37 @@
 
 		/**
 		 * Represents refinement choices for a node which is
+		 * in a container.
+		 * <p>
+		 * The node is a leaf node (i.e. no subnode can be added).
+		 *
+		 * @since 2.0
+		 */
+		interface InContainerLeafNodeCustomizableContextBuilder {
```

Comment:
Is it to prevent `inContainer()` being called twice? If so, it seems not really to pull its weight.